packages/core/data-transfer/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "extends": "tsconfig/base.json",
   "compilerOptions": {
-    "outDir": "dist"
+    "noEmit": true
```

Comment:
Can you explain this change so I'm sure I understand?

Reply:
I'm introducing a tsconfig.build.json which is in charge of emitting so I want to make sure the default one won't be used for that. and I did this to allow tsconfig to apply on test files as vscode only reads tsconfig.json it was ignored test folders